src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,162 @@
+package parser;
+
+import tasklist.*;
+import exception.*;
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+/**
+ * The Parser class is responsible for handling input commands.
+ */
+public class Parser {
+
+    /**
+     * Handles commands regarding tasks(ie. todos, deadlines, events).
+     * @param input The command input.
+     * @param newList The current TaskList.
+     * @throws DukeException if there is an error with the input or input is not given in the correct format.
+     */
+    public static void commandTask(String input, TaskList newList) throws DukeException {
+
+        String[] splitString = input.split(" ",2);
```

Comment:
It is an array of multiple items, so maybe a name indicating its plurality might be better? e.g. stringParts